ts/ui/menu/Menu.ts

```diff
@@ -1284,6 +1307,34 @@ export class Menu {
     this.rerender(STATE.COMPILED);
   }
 
+  /**
+   * @param {boolean} speech   Whether to use aria-label for Braille
+   */
+  protected setBrailleSpeech(speech: boolean) {
+    if (speech && this.settings.speech) {
+      Menu.loading++; // pretend we're loading, to suppress rerendering for each variable change
+      this.menu.pool.lookup('speech').setValue(false);
+      Menu.loading--;
```

Comment:
Could that be wrapped into the `noRerender` method?

Reply:
Yes it should.  I think it wasn't available in the develop branch when the top-braille branch (on which this was based) was created.  I'll make the change.